core/src/com/unciv/ui/tilegroups/CityButton.kt

```diff
@@ -160,9 +160,12 @@ class CityButton(val city: CityInfo, internal val tileGroup: WorldTileGroup, ski
     }
 
     private fun getPopulationGroup(showGrowth: Boolean): Group {
+        val growthGreen = Color(0.0f, 0.5f, 0.0f, 1.0f)
```

Comment:
This is much nicer, but there are conflicts (imo) with several civilizations:
![BadColor](https://user-images.githubusercontent.com/17114100/71830679-88bd4b00-30a7-11ea-828c-a0d655d6bfd0.png)
![BadColor2](https://user-images.githubusercontent.com/17114100/71830680-8955e180-30a7-11ea-8bba-b41a41973a5b.png)
![BadColor3](https://user-images.githubusercontent.com/17114100/71830681-8955e180-30a7-11ea-8039-40432a513858.png)

Maybe you can use the secondary color in case the city is growing?

Reply:
Fair enough, I was trying to replicate Civ 5, but yeah I agree with those civ colours it's a bit bad.

Do you want both the growth bar *and* the number to be the secondary colour, or just the number or just the growth bar?

Reply:
Don't know tbh. Maybe you can try and see how it looks? Or just wait for @yairm210 opinion

Reply:
Yeah, I'll see what @yairm210 says.

In my opinion, I think at least one of the bar or number (probably the bar) should stay the green colour, only because it may be initially confusing for new players if they see 2 numbers and a bar all the same colour, as they may not understand what differentiates them. Whereas if say the bar is green, it's the same colour as food, so the player *may* associate it with food at least.